components/application-authenticators/org.wso2.carbon.identity.application.authenticator.social/src/main/java/org/wso2/carbon/identity/application/authenticator/social/live/WindowsLiveOAuth2Authenticator.java

```diff
@@ -131,6 +131,15 @@ protected String getScope(String scope, Map<String, String> authenticatorPropert
     }
 
     /**
+     * We do not have a default claim dialect.
```

Comment:
This comment is not good enough. What does "We do not have a default claim dialect" mean as a method comment. A method level should explain what that method does. Also, you should mention what is returned by the method.